chrome/sunset.js

```diff
@@ -43,9 +34,7 @@ Sunset.start_ = new Date();  // TBD
  */
 Sunset.timeline_offsets_ = [
   0,
-  14,
-  28,
-  29
+  14
```

Comment:
15 should be listed here as well, as the last date here represents the uninstallation date.